extensions/jms/src/test/resources/mimeType/jms-mime-type-propagation.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:jmsn="http://www.mulesoft.org/schema/mule/jmsn"
+      xmlns:test="http://www.mulesoft.org/schema/mule/test"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+             http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd
+      http://www.mulesoft.org/schema/mule/jmsn http://www.mulesoft.org/schema/mule/jmsn/current/mule-jmsn.xsd">
+
+    <flow name="publisher">
+        <jmsn:publish config-ref="config" destination="${destination}"/>
+        <logger level="DEBUG" message="#[payload]"/>
+    </flow>
+
+    <flow name="publishConsumer">
+        <jmsn:publish-consume destination="${destination}">
+            <jmsn:message-builder >
+                <jmsn:reply-to destination="${destination}" destinationType="QUEUE"/>
+            </jmsn:message-builder>
+        </jmsn:publish-consume>
+    </flow>
+
+    <flow name="publishConsumer-with-override">
+        <jmsn:publish-consume destination="${destination}" contentType="text/json" encoding="US-ASCII">
+            <jmsn:message-builder >
```

Comment:
consider this message builder to be `showInDsl=false`, so that this is consistent with the changes that http recently went through